### DISCUSSION TITLE
Script setup

### DISCUSSION THREAD

**Discussion by prabhushan** [postalserver/postal#2718] [Help with using Postal]
Hi,
Right now i am setting postal server and setting up DNS, mail server and many other configurations using postal UI. Is there any possibility to use scripts like shell to setup DNS, mail server and inbound routes etc?

**Comment by prabhushan** [comment]
Any suggestions. Please help.

**Comment by willpower232** [comment]
Unless you can figure out the MySQL inserts and run those manually, I'm afraid not

**Reply by prabhushan** [reply]
Oh ok.. Thank you
